src/components/crossDevice/MobileConnected/style.css

```diff
@@ -8,21 +8,14 @@
   margin-bottom: 32*@unit;
 }
 
-.cancel {
-  line-height: 1.43;
-  font-size: @font-size-small;
-  width: 50*@unit-small;
-  margin: auto;
-  color: @color-text-dark;
-  text-decoration: none;
-  box-shadow: inset 0 -1*@unit 0 0 @color-primary-500;
-  cursor: pointer;
-}
-
 .help {
-  margin-bottom: 24*@unit;
+  /* placeholder to namespace this with CSS modules */
```

Comment:
Why this is needed, for E2E tests? Then I think it's better to use a data attribut, do you use something like `data-test` or `data-qa`?

Reply:
No, we don't use the data attributes. I do find it a bit unnecessary to have to do this but the UI tests fail to detect the element despite the classes being visually present in the UI. 
Will look into how it does with data attributes instead.